spec/text-modifiers.md

```diff
@@ -49,7 +49,7 @@ Example usage
 const canvas = document.createElement('canvas');
 const ctx = canvas.getContext('2d');
 
-ctx.textLetterSpacing = "3px";
+ctx.letterSpacing = "3px";
```

Comment:
In the HTML specs letterSpacing is defined as a double.
Does Chrome's implementation diverge?

Reply:
We will be changing the spec to reflect this. 
We believe it's better for letterSpacing/wordspacing to take string, so users can use different length units.

Reply:
Oh really? That's quite surprising, I can also see the argument that playing with numbers is more useful for canvas applications, `font` is already a pain point because we have to build a string from all weight, size and name.  
But the PR with this change will be a better place to discuss this.
Good to see it wasn't an oversight.